commodore/cluster.py

```diff
@@ -58,9 +58,9 @@ def _full_target(cluster, components, catalog):
     global_defaults = ['global.common',
                        f"global.distribution.{cluster_distro}",
                        f"global.cloud.{cloud_provider}"]
-    if 'region' in cluster_facts:
+    if 'region' in cluster_facts and cluster_facts['region']:
```

Comment:
It may make sense to ensure that the value of the optional fact is a string, and fail otherwise.

Reply:
This is already enforced by the API/K8s

Reply:
Never mind then :)